test/functional-api/pagination/organization.it-spec.ts

```diff
@@ -0,0 +1,330 @@
+      }
+    );
+  });
+
+  describe('Pagination without filter', () => {
+    // Arrange
+    test.each`
+      pagination       | result1                 | result2                 | organizationsCount
+      ${{ first: 11 }} | ${'org1-email@test.io'} | ${'org2-email@test.io'} | ${6}
+      ${{ last: 17 }}  | ${'org3-email@test.io'} | ${'org4-email@test.io'} | ${6}
+      ${{ first: 7 }}  | ${'org4-email@test.io'} | ${'org5-email@test.io'} | ${6}
+      ${{ first: 2 }}  | ${''}                   | ${'org1-email@test.io'} | ${2}
+    `(
+      'Quering: "$pagination", returns organizations: "$result1","$result2", and organizationsCount: "$organizationsCount" ',
+      async ({ pagination, result1, result2, organizationsCount }) => {
+        // Act
+
+        const request = await paginationFnOrganization(pagination);
+        const organizations =
+          request.body.data.organizationsPaginated.organization;
+
+        // Assert
+        expect(organizations).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              contactEmail: result1,
+            }),
+          ])
+        );
+        expect(organizations).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              contactEmail: result2,
+            }),
+          ])
+        );
+        expect(organizations).toHaveLength(organizationsCount);
+      }
+    );
+  });
+
+  //Skipped, until issue with organization query from clean DB is fixed
+  describe.skip('Pagination with cursors', () => {
+    // Rules
+    // Total: 5 users
+    // query first:2 -- > returns startCursor / endCursor - prevPage+false, nextPage-true
+
+    // 2 results
+
+    // ------
+    // query: first:1 / after: startCursor - returns second organization; hasNextPage: true, hasPreviousPage: false
+    // query: first:0 / after: startCursor - returns the 2 organizations from first batch; hasNextPage: true, hasPreviousPage: false - this one returns error (this case is importnant, as there is no way to get the first organization from first batch)
+    // query: first:1 / after: startCursor - returns second organization; hasNextPage: false, hasPreviousPage: true
+    // query: first:0 / after: startCursor - returns the 2 organizations from first batch; hasNextPage: false, hasPreviousPage: true - this one returns error (this case is importnant, as there is no way to get the first organization from second batch)
+    // const organizationData =
+    //   request.body.data.organizationsPaginated.organization;
+    // Arrange
+    let startCursor = '';
+    let endCursor = '';
+    beforeAll(async () => {
+      const request = await paginationFnOrganization({
+        first: 2,
+      });
+      startCursor =
+        request.body.data.organizationsPaginated.pageInfo.startCursor;
+      endCursor = request.body.data.organizationsPaginated.pageInfo.endCursor;
+
+      console.log(request.body.data.organizationsPaginated.organization);
```

Comment:
are the console logs necessary?

Reply:
Left them for debugging purposes, when we have the functionality implemented correctly. Do you want me to remove them now?

Reply:
No, that's alright